civihr_default_theme/scss/components/_header.scss

```diff
@@ -173,14 +172,16 @@
 
     &:last-child {
       > a {
-        border: none;
+        border: 0;
       }
     }
   }
 
-  @media(min-width: $screen-md-max) {
-    display: flex;
+  .menu-link-hr-admin {
+    order: 9999;
+  }
 
+  @media(min-width: $screen-md-max) {
```

Comment:
We mix `min-width` rule with `max` variable. Could you check that on this exact edge case (when the screen has **exactly** `$screen-md-max` width), it behaves as expected?

Reply:
Good point, fixed all such instances in the code.